.prettierrc

```diff
@@ -1,13 +1,5 @@
-printWidth: 120
-tabWidth: 2
-useTabs: false
-semi: true
-singleQuote: false
-quoteProps: "as-needed"
-jsxSingleQuote: false
-trailingComma: "all"
-bracketSpacing: true
-jsxBracketSameLine: false
-arrowParens: "always"
-parser: "typescript"
-endOfLine: "lf"
+{
+    "printWidth": 120,
+    "trailingComma": "all",
+    "parser": "typescript"
+}
```

Comment:
why change the format to JSON, it's easier to keep it all in plain text than JSON and also pretty convenient to have all the configuration explicitly mentioned (instead of depending on the changing defaults)

Reply:
I changed the format because Prettier expected a JSON object in VSCode, even though technically it's okay if it's not.  See attached.
<img width="303" alt="error" src="https://user-images.githubusercontent.com/2966898/79518106-4d69e980-80a4-11ea-9416-a609f7718305.png">

I removed the defaults because they are unnecessary. Also, Prettier documentation does not include all the configuration options too. https://prettier.io/docs/en/configuration.html

Reply:
This file is missing the end of file line (defined in [.editorconfig](https://github.com/infographicsgroup/sapera-components/blob/master/.editorconfig#L9)) so make sure that your vscode is configured to use the editor config plugin https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig to avoid such discrepancies between different machines...

I included the defaults for all the prettier configurations ([in the order they are defined in on prettier](https://prettier.io/docs/en/options.html)) to make it simple to override any of the options...
I omitted the "dangerous options", like `Range`, `Require pragma`, `Insert Pragma` (as unintentionally fiddling with those could make prettier skip running)
I also omitted the options that are not JS related like `Prose Wrap` (for markdown), `HTML Whitespace Sensitivity` as prettier should only be concerned with JS... leaving the options to format other files using other formats... 
That was the reason I added all the relative options (in the order of their definition on the website).

Reply:
Hey, sorry I missed your reply because the conversation is marked resolved. I only saw it because you reference it in this later [comment](https://github.com/infographicsgroup/sapera-components/pull/3#discussion_r410240724). 

Okay, I installed EditorConfig VSCode plugin, but it did not solve the problem so I will investigate further. 

Okay, a good reason to leave out the non-JS related Prettier options. As for `Range, Require pragma,` and `Insert Pragma` great thing Prettier omits them as well.